changelog.md

```diff
@@ -1,3 +1,24 @@
+# Version 0.4.0
+
+## Features
+
+* [METRONOME-190](https://jira.mesosphere.com/browse/METRONOME-190) Added authorized launch queue
```

Comment:
`authorized` is misleading here as I see. Suggesting that there was an unauthorized one already.